lib/utils/HDFDatabase.h

```diff
@@ -138,7 +152,19 @@ class HDFDatabase : public Database
         int* data,
         int nelements);
 
-    virtual int getDoubleArraySize(const std::string& key);
+    /**
+     * @brief Count the number of elements in
```

Comment:
This can be reformatted to use up to 80 characters per line.

Reply:
It seems that astyle does not change these documentation. I reformatted it manually.